AdvancedRocketry/blocks/LandingPad.php

```diff
@@ -1,39 +1,39 @@
 <?php
     /*  HOW TO USE THIS TEMPLATE
-        If you are unsure how to use this, please see any of the other pages for a reference. 
+        If you are unsure how to use this, please see any of the other pages for a reference.
 
-        Descriptions can include any HTML tags. Please link to other pages when you reference a block or item the first time. 
+        Descriptions can include any HTML tags. Please link to other pages when you reference a block or item the first time.
 
         <a href="Cables.php">data pipes</a>
 
         Please follow these naming conventions when submitting images for blocks:
-            
+
             PAGE IMAGE = nameOfBlock_demo.png
             SIDE BAR IMAGE = nameOfBlock_block.png
             ICON IMAGE FOR LIST PAGE = nameOfBlock_icon.png
-        
+
         Place images in the img folder. Be sure to update blocks/index.html with a link to your block page.
-    
-    */    
-    
+
+    */
+
     //NAME OF BLOCK
     $title = 'Docking Pad';
 
     $imageDemoName = 'dockingPad_demo.jpg';
-    
-    //USE <p></p> to make a new paragraph 
+
+    //USE <p></p> to make a new paragraph
     $description1 = '<p>The Docking Pad can be placed on a space station to specify a landing location for rockets on space stations.</p>
     <p>If all the Docking Pads on a space station are full or reserved, then an incoming rocket will land at the origin.</p>
-    <p>Infrastructure such as <a href="Hatches.php#rocket">Rocket loaders/unloaders</a>, fuel loaders, and <a href="RocketMonitoringStation.php">Rocket Monitoring Stations</a> can be connected to it using the <a href="../items/linker.php">linker</a>.</p>
+    <p>Infrastructure such as <a href="Hatches.php">Rocket Loaders/Unloaders</a>, fuel loaders, and <a href="RocketMonitoringStation.php">Rocket Monitoring Stations</a> can be connected to it using the <a href="../items/linker.php">linker</a>.</p>
```

Comment:
I've just noticed this is wrong, too... there's a separate page for loader/unloaders now, which is where this really should link to.